unlock-app/src/index.css

```diff
@@ -5,3 +5,22 @@
 body {
   @apply bg-ui-secondary-200;
 }
+
+.markdown p {
+  margin-bottom: 1em;
+}
+
+.markdown a {
+  text-decoration: underline;
+}
+
+.markdown ul {
+  margin-top: 1em;
+  margin-bottom: 1em;
+  list-style: disc outside none;
+}
+
+.markdown ul li {
+  margin-left: 2em;
+  display: list-item;
```

Comment:
You can actually use tailwind classes with `@apply`

Reply:
Please, tell me more!

Reply:
.markdown li {
@apply mb-2
}

So can create composeable classes like this. Mostly to remain consistent and reduce the lines of code.

Reply:
Also have you tried using "prose prose-md". This basically handles base typography for markdown. We use it on our static website for blog articles which are in markdown.

Reply:
Let me look!

Reply:
I could not make `@apply` work :/ Also looked at prose and prose-md but was confused :(